cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/ZookeeperLeaderFinder.java

```diff
@@ -43,7 +42,7 @@
  */
 public class ZookeeperLeaderFinder {
 
-    private static final Logger log = LogManager.getLogger(ZookeeperLeaderFinder.class);
+    private static final ReconciliationLogger LOGGER = ReconciliationLogger.create(ZookeeperLeaderFinder.class);
```

Comment:
Use the regular LOGGER here, the reconciliation logger does not seem to be needed. Although I wonder a bit why is reconciliation not propagated here and used here. This seem to be always bound to a particular reconciliation so it would make sense.

Reply:
I was not sure what Reconciliation object should be provided here https://github.com/strimzi/strimzi-kafka-operator/blob/9401f8be553c0fc174b4c53667ece17e23d3efcf/cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/ResourceOperatorSupplier.java#L98

Reply:
Well, I guess this might be one of the places where you cannot do it in constructor. But only in the methods called by the KafkaAssemblyOperator.